frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -45,6 +45,7 @@ export const cleanFunnelParams = (filters) => {
 const isStepsEmpty = (filters) => [...(filters.actions || []), ...(filters.events || [])].length === 0
 
 export const funnelLogic = kea({
+    key: (props) => props.dashboardItemId || 'some_funnel',
```

Comment:
this isn't actually the solution because it would then make the entire scene logic key-dependent. probably need to bring back funnelVizLogic

Reply:
I'm not sure why this is a bad thing? All of our other logics do it this way?

Reply:
So this is me commenting on my own commit here actually. The original PR didn't have this which was the problem - rather than creating individual logics for each funnel, all funnels were tied into the same logic, hence they had the same data, refreshed together, etc. 

But actually as of right now this commit is out of date. Essentially, what I think we need to do is bring back `funnelVizLogic`, but happy for you to take this as you wish. By getting rid of it and putting it all into one, we created a unified logic for all funnels. 

However, by adding the key here, which was my quick immediate attempt right after finding out this broke, we make it so that all of the logic here gets duplicated per funnel, including stuff like general scene logic such as `actionToUrl`. Hence, adding this line made the dashboards work fine, but broke the funnels tab.

Reply:
TL;DR: The problem isn't this line, it's where it is. But again, this is my own commit, wasn't in the original PR.